File: PieGallery/app/src/main/java/com/deadely/piegallery/view/info/presenter/InfoPresenter.java
package com.deadely.piegallery.view.info.presenter;

import com.deadely.piegallery.base.BasePresenter;
import com.deadely.piegallery.view.info.IInfoContract;

import javax.inject.Inject;

public class InfoPresenter extends BasePresenter<IInfoContract.View> implements IInfoContract.Presenter {
    @Inject
    public InfoPresenter() {
    }
}

File: PieGallery/app/src/main/java/com/deadely/piegallery/ui/info/presenter/InfoPresenter.kt
package com.deadely.piegallery.ui.info.presenter

import com.deadely.piegallery.base.BasePresenter
import com.deadely.piegallery.ui.info.InfoView
import moxy.InjectViewState
import javax.inject.Inject

@InjectViewState
class InfoPresenter @Inject constructor() : BasePresenter<InfoView>() {
    fun exit() {
        router.exit()
    }
}
